agent/engine/docker_task_engine.go

```diff
@@ -957,7 +955,7 @@ func (engine *DockerTaskEngine) buildCNIConfigFromTaskContainer(task *api.Task,
 	if !ok {
 		return nil, errors.New("engine: failed to find the pause container")
 	}
-	containerInspectOutput, err := engine.client.InspectContainer(pauseContainer.DockerName, inspectContainerTimeout)
+	containerInspectOutput, err := engine.client.InspectContainer(engine.ctx, pauseContainer.DockerName, inspectContainerTimeout)
```

Comment:
nit: this can be split into multiple lines to improve readability

Reply:
OK